src/resources/users/user-controllers.js

```diff
@@ -40,13 +58,20 @@ exports.login = async (req, res) => {
 
   try {
     const user = await Users.findByForLogin({ email });
-
+    console.log(user)
```

Comment:
Let's try to avoid console.logs in our next PR's. This week we start to get stricter with merging. Everything works as it should, I will merge this PR after your next commit. Great work Dom 🔥

Reply:
Good point, have removed them

Reply:
This PR contains my updates to register route also, should have specified this in main body